src/github/deployment.test.ts

```diff
@@ -1,177 +1,78 @@
-/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
Purely move the file to a proper location, and use the same name for unit testing.